### PR TITLE
Create a job to refresh tokens every 2 hours

### PR DESCRIPTION
**Pull request by jshaofuturewei**

### CODE REVIEW

File: globalscheduler/controllers/dispatcher/dispatcher_process.go [hunk 1]
```diff
@@ -260,3 +258,16 @@ func (p *Process) getHostIP(clusterName string) (string, error) {
 	p.clusterIpMap[clusterName] = cluster.Spec.IpAddress
 	return p.clusterIpMap[clusterName], nil
 }
+
+func (p *Process) refreshToken() {
+	for range time.Tick(time.Hour * 2) {
```
Comment (kimeunju108) [inline globalscheduler/controllers/dispatcher/dispatcher_process.go:263]: Do you have any reason why this is 2 hours? 
Isn't openstack's default value for token expiry one hour?

**Review by kimeunju108 — APPROVED** [review]

File: globalscheduler/controllers/dispatcher/dispatcher_process.go (hunk 1 above)
Comment (jshaofuturewei) [reply globalscheduler/controllers/dispatcher/dispatcher_process.go:263]: Token expired time are 12 hours.
2 hours are the time we test 1000 pods.

Comment (cathyhongzhang) [reply globalscheduler/controllers/dispatcher/dispatcher_process.go:263]: Could you make sure about the openstack token expiry?

**Review by cathyhongzhang — APPROVED** [review]
LGTM

**Pull request merged**